tests/std/tests/GH_002030_asan_annotate_vector/test.cpp

```diff
@@ -14,10 +14,20 @@
 
 using namespace std;
 
+#ifndef __SANITIZE_ADDRESS__
+#if defined(__clang__) && defined(__has_feature)
+#if __has_feature(address_sanitizer)
+#define __SANITIZE_ADDRESS_

```

Comment:
There appears to be a bug here - you are defining one trailing underscore, but I believe the intention was for two.